.github/workflows/ci.yml

```diff
@@ -32,5 +32,6 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python }}
-      - run: pip install -e .[lz4,zstd]
-      - run: python setup.py test
+      - run: python -m pip install lz4 zstandard pytest tox setuptools wheel
```

Comment:
I would ask to keep installing the xtarfile dependencies via pip for two reasons: (1) it's a basic sanity check that `pip install` works and (2) we don't duplicate the information of which modules are required for which xtarfile submodules in multiple places.

Reply:
Fixed maybe? in 909c480
Like I mentioned in another comment. I would prefer to keep tox since it's very handy. So I removed everything else and just use that for the CI. Please tell me if you prefer to just get rid of tox.

Perhaps you prefer to have the linter and various tests separate for some reason. Easy fix if you do.